data/json/items/ammo/8x40mm.json

```diff
@@ -0,0 +1,104 @@
+[
+  {
+    "id": "8mm_bootleg",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "bootleg 8x40mm",
+    "name_plural": "bootleg 8x40mm",
+    "description": "Bootleg duplicates of Rivtech 8x40mm caseless rounds.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "proportional": {
+      "price": 0.7,
+      "damage": 0.9,
+      "dispersion": 1.1
+    },
+    "extend": { "effects": [ "RECYCLED" ] },
+    "delete": { "effects": [ "NEVER_MISFIRES" ] }
+  },
+  {
+    "id": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40m",
+    "name_plural": "8x40m",
+    "//": "Cased ammo tends to be roughly $1/shot, more or less.  Rivtech ammo, being New and Proprietary and Expensive, $2-2.50 or so.",
+    "description": "8x40mm caseless rounds.  Proprietary ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "weight": 12,
+    "volume": 1,
+    "price": 12500,
+    "material": [ "plastic", "powder" ],
+    "symbol": "=",
+    "color": "pink",
+    "count": 40,
+    "stack_size": 40,
+    "ammo_type": "8x40mm",
+    "range": 42,
+    "damage": 42,
+    "pierce": 18,
+    "dispersion": 75,
+    "recoil": 400,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "id": "8mm_civilian",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40m sporting",
+    "name_plural": "8x40m sporting",
+    "description": "8x40mm caseless rounds, light sporting load sold for civilian use.  Proprietary ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "proportional": {
+      "price": 0.8,
+      "damage": 0.5,
+      "dispersion": 1.2,
+      "recoil": 0.5
+    },
+    "delete": { "effects": [ "NEVER_MISFIRES" ] }
+  },
+  {
+    "id": "8mm_fmj",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40mm FMJ",
+    "name_plural": "8x40mm FMJ",
+    "description": "8x40mm caseless rounds, full metal jacket.  Military grade ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "relative": {
+      "damage": -5,
+      "pierce": 10
+    }
+  },
+  {
+    "id": "8mm_hvp",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40mm HVP",
+    "name_plural": "8x40mm HVP",
+    "description": "8x40mm caseless rounds, saboted high-velocity penetrator type.  Advanced military grade, armor-piercing ammunition that fires a single depleted uranium flechette.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "price": 50000,
+    "count": 20,
+    "relative": {
+      "damage": 5,
```

Comment:
>Arguably too OP (comparable to 50 BMG) considering the guns that fire it are far superior. I wanted to increase overall availability of the Rivtech ammo in this PR and the HVP ammo became a balance point as a result.

I'm okay with tis change on paper, but I'd rather balance stuff happened in its own PR, or at least was properly marked instead of being mixed in among a bunch of other changes.

>Because you cannot mix ammo types within a magazine and splitting the bootleg spawns across a number of types could result in the frustrating scenario where you've looted a good total number of rounds but it's split across very similar (but not identical rounds).

The JSP is meant to be the *less crappy* handmade 8x40mm bullet. Couldn't we keep it as a player-craft only item at least?